powersimdata/data_access/data_access.py

```diff
@@ -93,6 +93,57 @@ def close(self):
         pass
 
 
+class LocalDataAccess(DataAccess):
+    """Interface to shared data volume"""
+
+    def __init__(self, root=None):
+        self.root = root if root else server_setup.DATA_ROOT_DIR
+
+    def copy_from(self, file_name, from_dir):
+        """Copy a file from data store to userspace.
+
+        :param str file_name: file name to copy.
+        :param str from_dir: data store directory to copy file from.
+        """
+        pass
+
+    def copy_to(self, file_name, to_dir, change_name_to=None):
+        """Copy a file from userspace to data store.
+
+        :param str file_name: file name to copy.
+        :param str to_dir: data store directory to copy file to.
+        :param str change_name_to: new name for file when copied to data store.
+        """
+        self.check_filename(file_name)
+        src = posixpath.join(server_setup.LOCAL_DIR, file_name)
+        file_name = file_name if change_name_to is None else change_name_to
+        dest = posixpath.join(self.root, to_dir, file_name)
+        self.check_file_exists(dest, should_exist=False)
+        self.copy(src, dest)
+        self.remove(src)
```

Comment:
Should it be called `move_to`. I guess this would apply to `DartaAccess` and `SSDataAccess`

Reply:
Yeah, that makes more sense.

Reply:
You can keep `copy_to` without the remove if we need it in the future and define `move_to` that will call `copy_to` + the `remove` method

Reply:
We also call `remove` in the ssh implementation, so we only need `move_to`, right? In which case the change would just be renaming all the usages of `copy_to`.

Reply:
Yeah, right now we only move files.